unicoremc/models.py

```diff
@@ -498,3 +539,16 @@ def destroy(self):
                 self.app_type, self.country)
 
         self.db_manager.destroy(self.app_type, self.country)
+
+
+@receiver(post_save, sender=Project)
+def publish_to_websocket(sender, instance, created, **kwargs):
```

Comment:
I dream of adding a hook that updates the health of an instance from *marathon* and updating the UI on Mission Control to reflect this